src/MoBi.Presentation/Tasks/Edit/EditTasksForSimulation.cs

```diff
@@ -93,7 +96,7 @@ private void exportAllResults(IMoBiSimulation simulation, DataRepository dataRep
 
       private void exportDataRepository(string fileName, IEnumerable<DataColumn> dataRepository)
       {
-         var dataTables = _dataRepositoryTask.ToDataTable(dataRepository, x => x.QuantityInfo.PathAsString);
+         var dataTables = _dataRepositoryTask.ToDataTable(dataRepository, x => x.QuantityInfo.PathAsString, _dimensionFactory.MergedDimensionFor);
```

Comment:
follow up change from yesterday. Potential bug here as well when exporting to excel